documentation/src/docs/asciidoc/release-notes/release-notes-5.10.0-M1.adoc

```diff
@@ -35,7 +35,15 @@ GitHub.
 
 ==== Deprecations and Breaking Changes
 
-* ❓
+* The `fixed` parallel execution strategy will allow the thread pool to be saturated by
+  default
+
+==== New Features and Improvements
```

Comment:
This document now contains 2 `New Features and Improvements` sections.

Please remove the unused section.